web-chat/src/App.tsx

```diff
@@ -9,13 +9,17 @@ import handleCommand from './command';
 import Room from './Room';
 import Waku from 'waku/waku';
 import { WakuContext } from './WakuContext';
+import {generate} from 'server-name-generator';
 
 export const ChatContentTopic = 'dingpu';
 
 export default function App() {
   let [stateMessages, setMessages] = useState<ChatMessage[]>([]);
   let [stateWaku, setWaku] = useState<Waku | undefined>(undefined);
   let [nick, setNick] = useState<string>('web-chat');
+  if(nick==='web-chat') {
+    setNick(generate());
+  }
```

Comment:
```suggestion
  let [nick, setNick] = useState<string>(generate());
```

Does this work?

Also, you need to run the linter: `cd web-chat; npm run fix` (the CI should pick on it).

Reply:
seems to work well, yes. i wasn't sure if that string had any side effect other than being the default, but you are right that's better.